gammapy/background/__init__.py

```diff
@@ -10,3 +10,4 @@
 from .template import *
 from .kernel import *
 from .models import *
+from .make import *
```

Comment:
Done. I moved all methods to either `Cube` in `cube.py` or `CubeBackgroundModel` in `models.py`. I left only the skeleton in `make_bg_cube_model` in `make.py`.